ml-proto/test/block.wast

```diff
@@ -200,7 +200,7 @@
 
 ;; TODO(stack): move this elsewhere
 (module (func $type-break-num-vs-void
-  (block (i32.const 66) (br 0))
+  (block (drop (i32.const 66)) (br 0))
```

Comment:
It seems ok for the AST apart from some constraint being assumed of 'explicit drop', but this constraint is overly restrictive as this code is valid with 'explicit drop'. I don't know your code, but might it just be a matter of relaxing a constraint, a simply matter to resolve in binaryen? This might not address all these cases, but perhaps a good number of them. If you could point me to the failing check then that might help me understand the challenge?